jjb/egeria/egeria-docker.yaml

```diff
@@ -0,0 +1,26 @@
+---
+- project:
+    name: egeria-configure
+    project-name: egeria-configure
+    project: egeria
+    mvn-settings: egeria-settings
+    archive-artifacts: ''
+    build-node: centos7-docker-4c-4g
+    cron: 'H 11 * * *'
+    mvn-snapshot-id: logs
+    container-public-registry: ''
```

Comment:
These values cannot be empty ... the job needs to know which registries is it going to pull/push to and from
container-public-registry
container-snapshot-registry

Reply:
this project is not using nexus for hosting docker images. We are using docker hub. The values are related to nexus.